src/components/Checkbox/tests/Checkbox.test.tsx

```diff
@@ -175,22 +130,6 @@ describe('<Checkbox />', () => {
         disabled: false,
       });
     });
-
-    it('can change values when disabled', () => {
-      const spy = jest.fn();
-      const checkbox = mountWithApp(
-        <Checkbox label="label" disabled onChange={spy} />,
-      );
-
-      checkbox.find('input')!.trigger('onKeyUp', {
-        keyCode: Key.Enter,
-      });
-      checkbox.setProps({checked: true});
-
-      expect(checkbox).toContainReactComponent('input', {
-        checked: true,
-      });
-    });
```

Comment:
Since we no longer manually handler the keyUp events, we no longer need to test this.